src/features/smartSearch/components/filters/PersonTags/index.tsx

```diff
@@ -51,6 +51,10 @@ const PersonTags = ({
       tags: [],
     });
 
+  const [selected, setSelected] = useState<CONDITION_OPERATOR | 'min_matching'>(
+    filter.config.condition
```

Comment:
The reason the condition is 'any' when editing is that `min_matching` also has the condition 'any'. The key to differentiating between 'any' and 'at least' is the presence of `min_matching`. It would look like this:
- at least : {condition:'any', min_matching:1}
- any : {condition:'any', min_matching:undefined}